utility/block.go

```diff
@@ -139,20 +138,23 @@ func (u *UtilityContext) UnstakeActorsThatAreReady() (err typesUtil.Error) {
 	for _, actorTypeInt32 := range typesUtil.ActorType_value {
 		var readyToUnstake []modules.IUnstakingActor
 		actorType := typesUtil.ActorType(actorTypeInt32)
-		poolName := ""
+		if actorType == typesUtil.ActorType_Undefined {
```

Comment:
Rather than having an `if` here, you can have a case for it (the first one) where you just `continue`.

Ditto bellow